ONSdigital/eq-survey-runner
# Add static translations

### What is the context of this PR?
- Added static translations for runner.

### How to review 
- Check that translations work on different types of surveys for static content.
- Check switching between languages.
- Check error pages.

### Checklist
